gravitee-apim-console-webui/src/management/api/creation/v4/api-creation-v4.component.spec.ts

```diff
@@ -917,10 +948,11 @@ describe('ApiCreationV4Component', () => {
     });
   });
 
-  function expectEntrypointsGetRequest(connectors: Partial<ConnectorListItem>[]) {
+  function expectEntrypointsGetRequest(connectors: Partial<ConnectorListItem>[], expands?: ListPluginsExpand[]) {
     const fullConnectors = connectors.map((partial) => fakeConnectorListItem(partial));
-
-    httpTestingController.expectOne({ url: `${CONSTANTS_TESTING.env.baseURL}/v4/entrypoints`, method: 'GET' }).flush(fullConnectors);
+    httpTestingController
+      .expectOne({ url: `${CONSTANTS_TESTING.env.baseURL}/v4/entrypoints${toQueryParams(expands)}` })
```

Comment:
suggestion: if we move the query param (as suggested in other comment) then we can directly add the `expand=icon` query param here and avoid to repeat it in every tests